code/modules/spells/spell_types/list_target/telepathy.dm

```diff
@@ -23,7 +23,7 @@
 		return
 
 	message = tgui_input_text(owner, "What do you wish to whisper to [cast_on]?", "[src]")
-	if(QDELETED(src) || QDELETED(owner) || QDELETED(cast_on) || !can_cast_spell())
+	if(QDELETED(src) || QDELETED(owner) || QDELETED(cast_on) || !can_cast_spell() || owner.stat >= UNCONSCIOUS)
```

Comment:
This lets unconscious people use the input, but not send the message. That's unintuitive. Is there a way you can set the spell flags to not let them use it while unconscious?

Reply:
When i tested it, it didnt let me activate the button while unconscious either but i can test it again to make sure.

Reply:
I just tested it again, the button itself to activate the spell is disabled when you go unconscious.

Reply:
Ok